errors/errors.go

```diff
@@ -80,6 +80,12 @@ func Aborted(format string, args ...any) error {
 		format, args...)
 }
 
+// constructs conflict error (HTTP: 503).
+func CannotReadFDB(format string, args ...any) error {
```

Comment:
It's better to call it `Unavailable` to be more generic and aligned with all other error names.
Also godoc expects comments of exported functions to start from function name. `Unavailable` in this case: `// Unavailable constructs service unavailable error (HTTP: 503)`

Reply:
@efirs thanks. I've made this change.